config/lib/authorization.js

```diff
@@ -0,0 +1,35 @@
+﻿'use strict';
+
+var config = require('../config'),
+  jwt = require('jsonwebtoken'),
+  lodash = require('lodash');
+
+var auth = {
+  signToken: signToken
+};
+
+// Export the token auth service
+module.exports = auth;
+
+// Sign the Token
+function signToken(user, options) {
```

Comment:
this parameter is there for anyone that wants to add/override any options coming from the env configuration.

See below where we're merging the two objects.